tools/package.json

```diff
@@ -10,11 +10,11 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "sync-request": "^2.1.0",
-    "tv4": "1.2.7",
-    "vscode-json-languageservice": "3.0.2",
     "mocha": "^2.3.3",
-    "mocha-each": "^1.1.0"
+    "mocha-each": "^1.1.0",
+    "sync-request": "^2.2.0",
+    "tv4": "^1.2.7",
+    "vscode-json-languageservice": "3.0.2"
```

Comment:
why these changes?

Reply:
I am not sure why that was changed.  It might have to do with the fact your tools don't work right out the box.  I did an NPM install of the required packages for your stuff and it probably modified those files.  I will do a checkout of the ones from the master branch.